meta-microservicebus-node/recipes-microservicebus-node/nodejs-dbus/nodejs-dbus_0.4.0.bb

```diff
@@ -1,78 +1,80 @@
+# Recipe created by recipetool
```

Comment:
Remove nodejs-dbus recipe, msb-core fetch this on demand.

Reply:
* Removed the **nodejs-dbus** recipe in https://github.com/axians/meta-microservicebus/pull/49/commits/fdad0fea0aeab1970fff0462c9923a79b2d68b34
* Edited **README** to remove occurrences of nodejs-dbus